src/arrayMethodPop.js

```diff
@@ -6,7 +6,13 @@
  */
 function applyCustomPop() {
   [].__proto__.pop2 = function() {
-    // write code here
+    if (this.length) {
+      let last = this[this.length - 1];
+      this.length--;
+      return last;
+    } else {
+      return undefined;
```

Comment:
You can just miss it if you do not return - you return undefined

Reply:
done